apps/src/sites/studio/pages/lessons/edit.js

```diff
@@ -10,7 +10,7 @@ import {Provider} from 'react-redux';
 import _ from 'lodash';
 
 //TODO Remove once we hook up real level data
-import {levelKeyList} from '@cdo/apps/lib/levelbuilder/lesson-editor/SampleActivitiesData';
+import {levelKeyList} from '../../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
```

Comment:
(Same comment as above)

Reply:
Good question! In this case, yes, we are including test data in a "user facing" page, which we are doing because the feature hasn't launched yet. In this case Dani did the front end work before the back end was ready to provide the needed data, so she was able to make progress by using test data to display on the page.

Reply:
Got it, thanks.  Also, just curious, why did we change from the "@cdo" import to the relative import?

Reply:
I think @cdo only works for /src directory and not the /test directory

Reply:
We don't have a good policy on `@cdo` imports vs relative imports, so right now it is up to personal preference. RubyMine works best with relative imports. I'm not sure about other IDEs. `@cdo` imports can be convenient when reaching from one part of the repo to another, such as between entry points (like this one) and a react component or between apps/test and a react component, to avoid long chains of `../../..`. Some people prefer `@cdo` imports in general because you don't have to change them when you copy/move import statements between files.